layouts/includes/session.html

```diff
@@ -19,35 +19,31 @@
         </div>
 
         <div class="col col-4">
-            {% if session.speakers.length == 0 %}
```

Comment:
@stephaniehobson this is the only thing I need review on in this PR. I removed the conditional in this section (and retabbed) in order to display a summary on sessions without speakers. Removing the conditional appears to have no ill effect, but I know you have a lot of code in expand.js that might expect something here.